apps/bfd-pipeline/bfd-pipeline-app/src/test/java/gov/cms/bfd/pipeline/app/AppConfigurationTest.java

```diff
@@ -0,0 +1,155 @@
+package gov.cms.bfd.pipeline.app;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import gov.cms.bfd.model.rif.RifFileType;
+import gov.cms.bfd.pipeline.ccw.rif.load.CcwRifLoadTestUtils;
+import gov.cms.bfd.sharedutils.config.ConfigException;
+import gov.cms.bfd.sharedutils.config.ConfigLoader;
+import io.micrometer.cloudwatch2.CloudWatchConfig;
+import io.micrometer.core.instrument.config.validate.ValidationException;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.time.Duration;
+import java.util.HashMap;
+import org.apache.commons.codec.binary.Hex;
+import org.junit.jupiter.api.Test;
+
+/** Unit tests for {@link AppConfiguration}. */
+public class AppConfigurationTest {
+  /**
```

Comment:
This is a straight up replacement for `AppConfigurationIT`.  We can use a simple unit test because the config doesn't need a forked process with environment variables to be tested anymore.  We can simply set up the variables in a map and pass that when creating the config.